spec/integration/api/conversations_controller_spec.rb

```diff
@@ -115,9 +115,9 @@
     it "returns all the user conversations" do
       get api_v1_conversations_path, params: {access_token: access_token}
       expect(response.status).to eq 200
-      returned_conversations = JSON.parse(response.body)
+      returned_conversations = JSON.parse(response.body)["data"]
       expect(returned_conversations.length).to eq 3
-      confirm_conversation_format(returned_conversations[0], @conversation, [auth.user, alice])
+      confirm_conversation_format(returned_conversations[1], @conversation, [auth.user, alice])
```

Comment:
Are you asking why it is the 0th instead of 1st?

Reply:
Yes

Reply:
I mean index not conting.

Reply:
It's probably an overly-fragile way to code the test I just manually looked at whichever conversation matched the GUID and used that one initially and then again now. It should probably search by GUID and pass that in so that if the order changes for whatever reason (like another test added a conversation somewhere to the test database) that it isn't broken by that.